cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/poller/context/VirtualMachineDeleteCheckerContext.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.cloudbreak.cloud.arm.poller.context;
+
+import com.sequenceiq.cloudbreak.cloud.arm.view.ArmCredentialView;
+
+public class VirtualMachineDeleteCheckerContext extends ArmStatusCheckerContext {
```

Comment:
Can we just use only one class instead of  VirtualMachineCheckerContext   + VirtualMachineDeleteCheckerContext